ArduPlane/Log.cpp

```diff
@@ -330,6 +366,15 @@ const struct LogStructure Plane::log_structure[] = {
       "PIQY", PID_FMT,  PID_LABELS, PID_UNITS, PID_MULTS },
     { LOG_PIQA_MSG, sizeof(log_PID),
       "PIQA", PID_FMT,  PID_LABELS, PID_UNITS, PID_MULTS },
+
+// @LoggerMessage: AETR
+// @Description: Value received by control surfaces of the aircraft
```

Comment:
This one needs a bit of help, I think.

If you're not running an `AETR` aircraft then this description is misleading. 

Perhaps, "normalised pre-mixer control surface outputs"?